test/Util.java

```diff
@@ -52,16 +53,20 @@ public class Util {
     private static final int SERVER_ALIVE_POLL_INTERVAL_MILLIS = 500;
     private static final int SERVER_ALIVE_POLL_MAX_RETRIES = SERVER_STARTUP_TIMEOUT_MILLIS / SERVER_ALIVE_POLL_INTERVAL_MILLIS;
 
-    public static File getArchivePath(int index) {
+    public static File getArchivePath(String flag) {
```

Comment:
We don't use indices anymore, so we pass the flag that the resource is seeking and use that to find the path.